lib/boot.js

```diff
@@ -118,13 +120,15 @@ let props = {
     auth
       .authorize(username, password)
       .then(user => {
+        resetStorageIfAccountChanged(username);
         if (!user.access_token) {
           return store.dispatch(resetAuth);
         }
 
         store.dispatch(setAccountName(username));
         store.dispatch(setAuthorized());
         localStorage.access_token = user.access_token;
+        token = user.access_token;
```

Comment:
It looks like we could probably avoid maintaining this `token` variable at file scope if we directly used `localStorage.access_token` as the single source of truth for our token.

Reply:
Created https://github.com/Automattic/simplenote-electron/issues/1026 to sort that out :)